### PR TITLE
add `push-labels` option for java sdk and add `--custom-lables` option for mount

### LINKED ISSUE #4245: Support --push-gateway-labels option
What would you like to be added:
Adding the "push-gateway-labels" option allows users to customize the labels for data push.
The format is:
```java
 <label1>:<value1>,<label2>:<value2>
```
For example:
serviceId : dsg1b24a5
When the "push-gateway" option is 10.67.207.138:6389, the data push interface URL is
http://10.67.207.138:6389/metric/job/juicefs/serviceId/dsg1b24a5

Why is this needed:
Add this option to allow users to customize the data push URL and push data to a custom monitoring system, improving JuiceFS's compatibility with Prometheus-related development products.

### PR DESCRIPTION
**Pull request by reAsOn2010**
close #4245

Does some defensive programming (check the format of parameter) needed ?

### CODE REVIEW

**Comment by zhijian-pro** [comment]
> Does some defensive programming (check the format of parameter) needed ?

Yes

**Comment by zhijian-pro** [comment]
The above code can only solve the problem of java sdk custom lable, go process also need to be added.
If you are interested, you can improve the section.
https://github.com/juicedata/juicefs/blob/6ef68130ebd6bc5b7081f30a5535bf8db47c64ae/cmd/mount.go#L162

**Comment by reAsOn2010** [comment]
OK, I searched the code base with pattern "push-gateway" then I only find the code in java sdk. I'll dive into the golang process code.

**Comment by zhijian-pro** [comment]
The previous [issue](https://github.com/juicedata/juicefs/issues/4245) was a special case, and we should support custom lables, whether push(push-gateway) or pull(prometheus).

File: docs/en/deployment/hadoop_java_sdk.md [hunk 1]
```diff
@@ -204,6 +204,7 @@ Please refer to the following table to set the relevant parameters of the JuiceF
 | `juicefs.push-auth`       |               | [Prometheus basic auth](https://prometheus.io/docs/guides/basic-auth) information, format is `<username>:<password>`.                                                       |
 | `juicefs.push-graphite`   |               | [Graphite](https://graphiteapp.org) address, format is `<host>:<port>`.                                                                                                     |
 | `juicefs.push-interval`   | 10            | Metric push interval (in seconds)                                                                                                                                           |
+| `juicefs.push-labels`     |               | Metric labels, format is `<key1>:<value1>,<key2>:<value2>`.                                    |
```
Comment (zhijian-pro) [inline docs/en/deployment/hadoop_java_sdk.md:207]: `key1:value1,key2:value2`  
This format is simple and straightforward and will not be misinterpreted

Comment (reAsOn2010) [reply docs/en/deployment/hadoop_java_sdk.md:207]: so you mean just delete this part of doc?

Comment (zhijian-pro) [reply docs/en/deployment/hadoop_java_sdk.md:207]: just replace `<key1>:<value1>,<key2>:<value2>` to `key1:value1,key2:value2`

Comment (zhijian-pro) [reply docs/en/deployment/hadoop_java_sdk.md:207]: Like this.
image

**Comment by reAsOn2010** [comment]
Is this PR causing test failure?

**Comment by zhijian-pro** [comment]
It doesn't matter if `mutate-test` doesn't pass.

**Comment by zhijian-pro** [comment]
LGTM!

**Review by zhijian-pro — APPROVED** [review]

**Pull request merged**